pymclevel/leveldbpocket.py

```diff
@@ -348,17 +348,7 @@ def _readSubChunk_1plus(self, cx, cz, y, readOptions=None, key=None):
                     write_dump("!!! An unhandled error occured when loading sub-chunk (%s, %s, %s) terrain.\n" % (cx, cz, y))
                     write_dump("%s" % e)
 
-            try:
-                tile_entities = db.Get(rop, key + "\x31")
-            except RuntimeError:
-                tile_entities = None
-
-            try:
-                entities = db.Get(rop, key + "\x32")
-            except RuntimeError:
-                entities = None
-
-        return terrain, tile_entities, entities
+        return terrain
```

Comment:
Why did you remove that part and made it outside? It makes more sense that the "read_chunk" function will give you all the details and that it won't be outside